File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcControl/IfcCostItem/IfcCostItem.cs
public class IfcCostItem : IfcControl
{
    public IfcCostItemTypeEnum PredefinedType { get; set; }
    public List<IfcCostValue> CostValues { get; set; }
    public List<IfcPhysicalQuantity> CostQuantities { get; set; }
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcControl/IfcCostItem/IfcCostItem.py
from typing import List

class IfcCostItem(IfcControl):
    def __init__(self):
        self.PredefinedType: IfcCostItemTypeEnum = None
        self.CostValues: List[IfcCostValue] = None
        self.CostQuantities: List[IfcPhysicalQuantity] = None
